src/components/v5/common/WidgetCards/partials/WidgetSubTitle.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { type FC } from 'react';
+
+interface SubTitleProps {
+  title: React.ReactNode;
+}
+
+export const WidgetSubTitle: FC<SubTitleProps> = ({ title }) => (
+  <h3 className="py-1 heading-4">{title}</h3>
```

Comment:
This should maybe be a `<h4>` as currently the title and subtitle are both `<h3>`s.